### DISCUSSION TITLE
Is there a way to separate vocals from two singers?

### DISCUSSION THREAD

**Discussion by ytt246** [Anjok07/ultimatevocalremovergui#885] [Q&A]
I've extracted [this](https://whyp.it/tracks/129878/two-vocals?token=kIulO) vocals-only audio from the song and is trying to separate the two singers. Is it possible to do this? I've seen some recommending 5_HP and 6_HP-Karaoke models but they don't seem to do much. Are there any specific settings I need to use?

**Comment by Limuu1** [comment]
Im also looking for this,

 for now-- unless vocal1, vocal2, or even vocal3 are stacked or layered on top of each other an alternative i chose best is to just cut the clips manually or try to hide it by defusing the frequency of the voice your trying to block. (given that they are on different pitch) Ex. Deep for male singer Or high Pitch for Female. 

if its a removal of a reverb or like an echo type thing then idk.

**Comment by Karamel69** [comment]
I m going to look after that too, with a singer and a chorus group at the end of a song, uvr Makes so spectacular things that I though that it is possible, i don't find until now,
did from France

**Comment by Karamel69** [comment]
I found something
with UVR-MDX-NET karaoké et karaoké 2
with additional settings vocal splitter  select save... and enable...

with karaoké one voice
with karaoké 2 the back vocals

did
